django/template/backends/django.py

```diff
@@ -59,10 +60,15 @@ def __init__(self, template, backend):
     def origin(self):
         return self.template.origin
 
-    def render(self, context=None, request=None):
+    def render(self, context=None, request=None, stream=False):
         context = make_context(context, request)
+        if stream:
+            return self._stream(context)
+        return mark_safe(''.join(self._stream(context)))
```

Comment:
Can't you just return `self.template.render(context, stream=stream)`?

I'd like to avoid the hardcoded `stream=True` then `''.join()` dance.